pkg/api/v1/types.go

```diff
@@ -460,6 +462,8 @@ type PersistentVolumeClaimStatus struct {
 	AccessModes []PersistentVolumeAccessMode `json:"accessModes,omitempty"`
 	// Represents the actual resources of the underlying volume.
 	Capacity ResourceList `json:"capacity,omitempty"`
+	// Current service state of claim.
+	Conditions []PersistentVolumeClaimCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"type"`
```

Comment:
@bgrant0607 @thockin what is the merge attributes? I cargo culted this from the PodCondition. Please let me know what is appropriate here.